multiqc/modules/hicpro/hicpro.py

```diff
@@ -0,0 +1,386 @@
+            anchor = 'hicpro-filtering',
+            description = 'Filtering of read pairs.',
+            helptext = '''
+            Aligned read pairs are filtered to select valid 3C products from two 
+            different restriction fragments. Read pairs coming from the same fragments, 
+            such as self-ligation or unligated (danging-end) fragments, are discarded.
+            Ligation products involving neighboring restriction fragment (religation) are also discarded.
+            Finaly, as the ligation should be a random process, valid read pairs from all 
+            orientations (R=Reverse, F=forward) are expected to be observed at the same proportion.''',
+            plot = self.hicpro_filtering_chart()
+        )
+
+        self.add_section (
+            name = 'Contact Statistics',
+            anchor = 'hicpro-rmdup',
+            description = 'Contacts statistics after duplicates removal.',
+            helptext = '''
+            Description of contact frequency after duplicates removal. Intra-chromosomal 
+            (*cis*) interaction are expected to be more frequent than inter-chromosomal contacts (*trans*)''',
+            plot = self.hicpro_contact_chart()
+        )
+
+        allele_plot = self.hicpro_as_chart()
+        if allele_plot:
+            self.add_section (
+                name = 'Allele-specific analysis',
+                anchor = 'hicpro-asan',
+                description = 'Assignment of valid interactions (afer duplicates removal) to parental alleles.',
+                helptext = '''
+                Description of allele-specific contacts. Valid interactions (0-1 and 1-1, resp. 0-2, 2-2) 
+                are used to build the genome1 (resp. genome2) parental maps.''',
+                plot = self.hicpro_as_chart()
+            )
+            
+
+
+    def parse_hicpro_stats(self, f):
+        """ Parse a HiC-Pro stat file """
+        s_name = self.clean_s_name(os.path.basename(f['root']), os.path.dirname(f['root']))
+
+        if s_name not in self.hicpro_data.keys():
+            self.hicpro_data[s_name] = {}
+
+        self.add_data_source(f, s_name)
+        for l in f['f'].splitlines():
+            if not l.startswith('#'):
+                s = l.split("\t")
+                if s[0] in self.hicpro_data[s_name]:
+                    log.debug("Duplicated keys found! Overwriting: {}".format(s[0]))
+                self.hicpro_data[s_name][s[0]] = int(s[1])
+
+    def hicpro_stats_table(self):
+         """ Add HiC-Pro stats to the general stats table """
+         headers = OrderedDict()
+         headers['total_R1'] = {
+             'title': 'Total',
+             'description' : 'Total Number of Read Pairs',
+             'min' : '0',
+             'scale' : 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key' : 'read_count',
+             'hidden': False
+         }
+
+         headers['mapped_R1'] = {
```

Comment:
Table is looking much cleaner, thanks! I still think that these columns are a little unnecessary as there's a plot right underneath with the same numbers. But if you think that they should be there then it's fine to keep them.